merchants/verkkokauppa/order/requests.py

```diff
@@ -15,17 +15,19 @@
 def create_order(params: CreateOrderParams, post=_post) -> Order:
     try:
         response = post(
-            url=settings.VERKKOKAUPPA_ORDER_API_URL,
+            url=settings.VERKKOKAUPPA_ORDER_API_URL + "/",
             json=params.to_json(),
             headers={"api-key": settings.VERKKOKAUPPA_API_KEY},
             timeout=REQUEST_TIMEOUT_SECONDS,
         )
-        json = response.json()
+        json = (
+            response.json()
+        )  # TODO: In case of 501 there is no JSON response. Check response headers
```

Comment:
could a ticket numer

Reply:
Added TODO comment for myself but forgot to implement the DO part 😄 I'll fix this.

Reply:
Fixed 👍